examples/rl_gsm8k/utils.py

```diff
@@ -354,8 +359,19 @@ def launch_training(
         ValueError: If no GPUs are available
         RuntimeError: If training process fails
     """
+    # environment variables
+    GLOBAL_RANK = int(os.environ.get("RANK", 0))
+    MASTER_PORT = int(os.environ.get("MASTER_PORT"))
+    MASTER_ADDRESS = os.environ.get("MASTER_ADDR")
+    # this is same as number_of_replicas
+    WORLD_SIZE = int(os.environ.get("WORLD_SIZE", 2))
+
     # Check GPU availability
     num_gpus = torch.cuda.device_count()
+    print('###############################')
```

Comment:
please use logger.info, and maybe smth like messages, "I'm rank X, training on Y GPU"